demos/smart_classroom_demo/cpp_gapi/smart_classroom_demo.hpp

```diff
@@ -0,0 +1,148 @@
+// Copyright (C) 2018-2019 Intel Corporation
```

Comment:
seems this file should be renamed to smart_classroom_demo_gapi.hpp

Reply:
Changed.